static/src/javascripts/projects/common/modules/experiments/tests/live-blog-chrome-notifications-prod.js

```diff
@@ -7,7 +7,7 @@ define([
 ) {
     return function () {
 
-        this.id = 'LiveBlogChromeNotificationsProd';
+        this.id = 'LiveBlogChromeNotificationsProd2';
         this.start = '2016-06-08';
```

Comment:
I think you'll also need to change this line:
https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/enhanced/liveblog.js#L71